serverlessworkflow/modules/ROOT/pages/integrations/custom-functions-knative.adoc

```diff
@@ -0,0 +1,335 @@
+= Invoking Knative services from {context} custom functions
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the Knative service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, knative
+
+This document describes how to call Knative services using {context} custom functions. The procedure described in this document is based on the link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative[`serverless-workflow-custom-function-knative`] example application.
+
+The testing procedure in this document is based on the serverless-workflow-examples/serverless-workflow-custom-function-knative example application. You can access this example application in the link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative[Kogito Examples] GitHub repository.
```

Comment:
Seems like you are repeating yourself here :)

Reply:
I removed the second one ;)